app/models/relationship.rb

```diff
@@ -0,0 +1,4 @@
+class Relationship < ApplicationRecord
+  belongs_to :follower, class_name: "User"
+  belongs_to :followeb, class_name: "User"
```

Comment:
フォローしている人 なら 下記の方が一般的かと！
```suggestion
  belongs_to :following, class_name: "User"
```

Reply:
following に変更したことによる副作用は確認済みでしょうか?
テスト結果のスクリーンショットもあわせて添付お願いします。

Reply:
followingとfollowerの組み合わせが一般的かなと思います。